contribute.json

```diff
@@ -3,27 +3,20 @@
     "description": "The app powering MDN Web Docs.",
     "repository": {
         "url": "https://github.com/mdn/kuma",
-        "license": "MPL2",
+        "license": "MPL-2.0",
         "tests": "https://travis-ci.com/mdn/kuma"
     },
     "participate": {
         "home": "https://wiki.mozilla.org/MDN",
         "docs": "https://kuma.readthedocs.io/",
         "mailing-list": "https://discourse.mozilla.org/c/mdn",
-        "irc": "irc://irc.mozilla.org/#mdndev",
-        "irc-contacts": [
-            "djf",
-            "espressive",
-            "jwhitlock",
-            "peterbe",
-            "rjohnson"
-        ]
+        "chat": {
+            "url": "https://mozilla.slack.com/archives/CLTBV3W9X"
```

Comment:
How I did that
<img width="484" alt="Screen Shot 2020-05-07 at 11 03 46 AM" src="https://user-images.githubusercontent.com/26739/81310718-79004400-9052-11ea-9984-3dd6332bff54.png">